db/dictionary_es.csv

```diff
@@ -22,97 +22,97 @@ events,eventos,events,
 objects,objetos o artículos,objects or items,
 Deluge,Diluvio,Deluge,
 Adam,Adán,Adam,Luke 3:38
-Seth,Seth,Seth,
+Seth,Set,Seth,
 Enosh,Enós,Enosh,
-Kenan,Kenán,Kenan,
-Mahalalel,Mahalalel,Mahalalel,
+Kenan,Cainán,Kenan,
+Mahalalel,Mahalaleel,Mahalalel,
 Jared,Jared,Jared,
 Enoch,Enoc,Enoch,
 Methuselah,Matusalén,Methuselah,Luke 3:37
 Lamech,Lamec,Lamech,
 Noah,Noé,Noah,
 Shem,Sem,Shem,
-Arpachshad,Ar·pajshad,Arpachshad,Luke 3:36
-Shelah,Sela,Shelah,
+Arpachshad,Arpaksad,Arpachshad,Luke 3:36
+Shelah,Selá,Shelah,
 Eber,Éber,Eber,
 Peleg,Péleg,Peleg,
-Reu,Reu,Reu,
+Reu,Reú,Reu,
 Serug,Serug,Serug,Luke 3:35
 Nahor,Nacor,Nahor,
 Terah,Taré,Terah,
 Abraham,Abrahán,Abraham,
-Isaac,isaac,Isaac,
-Jacob,jacob,Jacob,Luke 3:34
+Isaac,Isaac,Isaac,
+Jacob,Jacob,Jacob,Luke 3:34

```

Comment:
Names taken from Luke 3

Reply:
Weird how the lower case names slipped by

Reply:
Seems unique to Spanish